xla/stream_executor/command_buffer.h

```diff
@@ -119,6 +119,11 @@ class CommandBuffer {
   // for updating and finalizing conditional command buffers.
   tsl::Status If(DeviceMemory<bool> pred, Builder then_builder);
 
+  // Adds a device memory allocation node to the command buffer.
+  // The VA allocation is performed during graph creation, and it does not
```

Comment:
Documentation is a bit CUDA graph specific, can you make it more abstract, VA and graph creation is an implemetnation details for CUDA